core/cmd/app.go

```diff
@@ -588,6 +588,7 @@ func NewApp(client *Client) *cli.App {
 
 				keysCommand("Solana", NewSolanaKeysClient(client)),
 				keysCommand("Terra", NewTerraKeysClient(client)),
+				keysCommand("Starknet", NewStarknetKeysClient(client)),
```

Comment:
It's "StarkNet" not "Starknet" :)

```suggestion
				keysCommand("StarkNet", NewStarkNetKeysClient(client)),
```

Reply:
renamed functions according to this:
`Starknet` => `StarkNet`
`starknet` => `starknet` (if first is lowercase, all of it is lowercase otherwise if looks weird in a parameter named `starkNetKey` - looks like `Net` is separate)